tools/spec-gen-sdk/package.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "spec-gen-sdk",
```

Comment:
Will this package be published to npmjs?  If so, I believe our convention is to use prefix `@azure-tools`.  Example:

https://github.com/Azure/azure-sdk-tools/blob/main/tools/js-sdk-release-tools/package.json

Reply:
Yes, I updated it along with the pipeline definitions.